.travis.yml

```diff
@@ -22,11 +25,11 @@ jobs:
       before_install: pip install -e .
       install: bash .travis.sh install-tools
       before_script:
-        - minikube config set cpus 4 && minikube config set memory 8192 && minikube config set vm-driver none && minikube config set kubernetes-version "${K8_VERSION}" 
+        - minikube config set cpus 4 && minikube config set memory 8192 && minikube config set vm-driver none && minikube config set kubernetes-version "${K8_VERSION}"
         - sudo minikube start
       script:
-        - ckan-cloud-operator cluster initialize --interactive --cluster-provider=minikube
-        - sleep 60 && kubectl get ns && kubectl get pods -n ckan-cloud 
+        - ckan-cloud-operator cluster initialize --cluster-provider=minikube || travis_terminate 1;
```

Comment:
We should probably keep the `--interactive`, as there are a few code paths that check it explicitly.